WordPressVIPMinimum/Sniffs/Compatibility/RestrictedCacheGroupSniff.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * WordPressVIPMinimum Coding Standard.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+
+namespace WordPressVIPMinimum\Sniffs\Compatibility;
+
+use WordPressCS\WordPress\AbstractFunctionParameterSniff;
+
+/**
+ * This sniff checks if global memcached group names are being used.
```

Comment:
Could this description be made more "descriptive" ? What are "global" memcached group names ?

```suggestion
 * Check if cache group names are being used which are already in use by the wp-memcached plugin which is always enabled on the VIP platform.
```

(not sure if the above is correct, but just giving you some idea of what I mean by my remark)